hcx-core/hcx-common/src/main/java/org/swasth/common/dto/Response.java

```diff
@@ -4,6 +4,7 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.HashMap;
+import java.util.LinkedHashMap;
```

Comment:
Where are we using the LinkedHashMap?

Reply:
This is an unused import statement, removed id.